src/UI/Actions.hs

```diff
@@ -807,12 +825,12 @@ trySendAndCatch l' m s = do
 sanitizeMail :: MIMEMessage -> MIMEMessage
 sanitizeMail = over (attachments . headers . contentDisposition . filename) (T.pack . takeFileName . T.unpack)
 
-initialCompose :: Compose
-initialCompose =
+initialCompose :: [Mailbox] -> Compose
+initialCompose mailboxes =
   let mail = B.empty
   in Compose
         mail
-        (E.editorText ComposeFrom (Just 1) "")
+        (E.editorText ComposeFrom (Just 1) (decodeLenient $ renderMailboxes mailboxes))
```

Comment:
`renderMailboxes` will probably render non-ASCII content using encoded-word, which is probably not what you want.
More likely you want to render the `Mailbox`es directly as `Text`?  (You would need to write that render function).

Reply:
That reminds me: https://github.com/purebred-mua/purebred-email/pull/15#discussion_r224306043 I didn't implement it. Perhaps I'll do that first and think of a pretty printing function.

Reply:
Hm... you mentioned in purebred-mua/purebred-email#15 that you're working on encoded-word support for address headers. I suppose I'll wait until this is merged before I can continue here?

Reply:
@romanofski I am working on that, but I don't think you're blocked.  You should focus on pretty printing and parsing the `Address` (and related types, if necessary) to/from `Text`.  The encoded-word work focuses on the 7bit-compatible conversion to/from ByteString for use directly within RFC822/RFC5322 messages.

Reply:
Hm... there is something missing I don't understand.

At this point, I go from `Mailbox` to `ByteString` to `Text`. Your proposal is to pretty print it from `Mailbox` to `Text`. That seems to be the easy part.
What I'm missing is the parser. Where would a parser which parses a `Text` instead of a `ByteString` be different? If I currently try to stick a:

```Haskell
"\"Róman Joost\" <roman@host>" :: ByteString
```
into the parser I get:
```
Left "\": Failed reading: satisfy"
```
Understood, because the display name contains non-ascii characters which will have to be encoded. In order to make the parser work with Text, I'd basically replace the necessary small bits to parse a `Text` instead of a `ByteString`. That sounds like duplication? Where is my mistake?

Reply:
There is no mistake; we'll have to duplicate some stuff.

Reply:
I've went down the rabbit hole and implemented most of the parser actions, which were rarely changes necessary apart from the different type functions. Perhaps we'll have to expose this module with some reshuffling? Something like:

```Haskell
-- the current parser
Data.RFC5322.ByteString.Address
-- and
Data.RFC5322.Text.Address
```